superset/assets/src/visualizations/big_number.js

```diff
@@ -93,23 +93,24 @@ function bigNumberVis(slice, payload) {
 
   if (fd.viz_type === 'big_number') {
```

Comment:
should this file still be around?

Reply:
Good catch. The merge must did something wrong.